### DISCUSSION TITLE
Synth surgery changes bug thread

### DISCUSSION THREAD

**Discussion by Shroopy** [Bubberstation/Bubberstation#2106] [codebase discussion]
Checkmark means that I think the problem is fixed.

Reproduced issues (I tested this and found the bug):
- Disassemble is not available for augmented podpeople ✅
- Organic brain surgery can not be performed on augmented people ✅

Unreproduced reported issues (I tested this and did not find the bug):
- Disassemble is not available for synths

Reported issues (I have not tested this yet)
- "The naming schema is also very confusing, and leaves us guessing at what does what." ✅
- "The system shock dampening surgery doesn't seem to do anything. Both self surgery and being performed by someone else it does nothing to stop you from getting shocked, or reduce the amount you get shocked to my eyes. Though i dont know much about the organic counterpart."
- "I don't know too much about the circumstances behind it, but I recently ran into an issue regarding being unable to reattach a limb someone lost, neither jamming it in nor draping their missing arm for surgery worked. The drape just gave the message that there wasn't any surgery to do. They were an anthromorph and a synthetic, if it's any help.

**Comment by Hislyna** [comment]
From my own testing, you cannot start self surgery with a bedsheet, only drapes! Not sure if intentional or not. You also now cannot do self surgery standing up like you could before. Also unsure if intentional or not.

**Reply by Shroopy** [reply]
I'll look into the bedsheet issue.
Being able to do surgery standing up depends on the specific surgery. Are there specific surgeries that should be doable standing up?

**Reply by Hislyna** [reply]
Organ Manipulation and Chassis manipulation are the ones that have always worked standing!

> I'll look into the bedsheet issue. Being able to do surgery standing up depends on the specific surgery. Are there specific surgeries that should be doable standing up?
